modules/gapi/src/api/grunarg.cpp

```diff
@@ -0,0 +1,33 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#include "precomp.hpp"
+#include <opencv2/gapi/garg.hpp>
+
+cv::GRunArg::GRunArg() {
```

Comment:
`=default` ?

Reply:
I prefer to keep constructor in `.cpp` for exported classes, even if those are empty.